.github/workflows/compute_version.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+VERSION_FILE="$SCRIPT_DIR/../../version"
```

Comment:
I'd keep the script config-agnostic and pass the file path via argument or env. 
Or even pass the version string to the script directly.

Reply:
Fixed in [f08521](https://github.com/KIT-MRT/arbitration_graphs/pull/85/commits/f08521a5601fc0d6bd582e124c3f95227973feab)